web-basics-browser-security-model/activities/cockroach/sol/solution

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+PORT=8080
+
+if [[ $1 == "local" ]]
+then
+    remote='http://127.0.0.1:'$PORT
```

Comment:
Again I'm having trouble with the variable name `remote`. A better one would be `url` or `address`